16.0/test/test-app/src/main/java/Test2.bak

```diff
@@ -0,0 +1,6 @@
+// a dummy file to force an actual javac in the maven build
+public class Test2 {
```

Comment:
Was this intended to be added?

Reply:
No. I lazily cloned the WF 15 and modified bits for WF 16. That file is present elsewhere so I've removed it from the entire codebase in 9a9b38ba0e37848ca4bcc030d87f5c59a190e842.